backend/pom.xml

```diff
@@ -180,7 +175,7 @@
     <resources>
       <!-- add basic application properties -->
       <resource>
-          <directory>${project.basedir}/src/main/resources</directory>
+          <directory>backend/src/main/resources</directory>
```

Comment:
please undo as the variable resolves to the backend folder.

Reply:
Done!